### PR TITLE
Refactor to java script classes

### PR DESCRIPTION
**Pull request by Cluab**
Completed Project requirements

- > Use JavaScript classes instead of objects and arrays.
- > Create class methods to add and remove books.
- > Maintain all of the original functionality of the application (i.e. no bugs were introduced while refactoring).
- > Don't use any library or framework, use only plain JavaScript.
- > Add CSS styles to the application to make it match the wireframe.

### CODE REVIEW

File: classScript.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    removeBtn.addEventListener('click', (event) => {
+    // Remove item from DOM
+      event.target.parentElement.remove();
+      // Remove item from booksList Array
+      booksList.splice(booksListIndex, 1);
+      // Remove iem from localStorage
+      localStorage.setItem('list', JSON.stringify(booksList));
+    });
+  }
+}
```
Comment (AshakaE) [inline classScript.js:47]: - [ ] Kindly intialize your booklist inside of the class as it is a requirement for this milestone

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @Mujeeb4582 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good job on the pull request 👍 
- Git flow ✅ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: classScript.js (hunk 1 above)
Comment (Mujeeb4582) [reply classScript.js:47]: @AshakaE Linters didn't allow us to create two classes. 
Kindly explain, a booklist means an array that stored the objects.

Comment (AshakaE) [reply classScript.js:47]: You can initialize the booklist array in the class constructor

**Pull request closed without merging**